packages/react-components/react-search-preview/stories/SearchBox/SearchBoxAppearance.stories.tsx

```diff
@@ -36,17 +24,11 @@ export const Appearance = () => {
         <SearchBox appearance="underline" />
       </Field>
 
-      <Field
-        className={mergeClasses(styles.fieldWrapper, styles.filledLighter)}
-        label={{ children: 'Filled lighter appearance', className: styles.filledLighterLabel }}
-      >
+      <Field className={styles.fieldWrapper} label="Filled lighter appearance">
         <SearchBox appearance="filled-lighter" />
       </Field>
 
-      <Field
-        className={mergeClasses(styles.fieldWrapper, styles.filledDarker)}
-        label={{ children: 'Filled darker appearance', className: styles.filledDarkerLabel }}
-      >
+      <Field className={styles.fieldWrapper} label="Filled darker appearance">
         <SearchBox appearance="filled-darker" />
       </Field>
```

Comment:
Was removing the background intentional? We should have that background as shown in https://react.fluentui.dev/?path=/docs/components-textarea--default#appearance, right?

Reply:
ah yes, it was! i forgot to move this PR to draft, it's something i'm looking into with design